src/olympia/promoted/signals.py

```diff
@@ -68,11 +68,13 @@ def promoted_approval_to_promoted_addon_version(
     instance: PromotedApproval,
 ):
     if signal == models.signals.post_save:
-        PromotedAddonVersion.objects.update_or_create(
-            version=instance.version,
-            promoted_group=PromotedGroup.objects.get(group_id=instance.group_id),
-            application_id=instance.application_id,
-        )
+        # Only create PromotedAddonVersion records if application_id is not None
```

Comment:
When `PromotedApproval`'s `application_id` is `None`, that would represent being approved for both apps. It should be creating `PromotedAddonVersion`s for each app in `APP_USAGE`.